lib/Chat/ChatManager.php

```diff
@@ -400,6 +400,30 @@ public function getHistory(Room $chat, int $offset, int $limit, bool $includeLas
 		return $this->commentsManager->getForObjectSince('chat', (string) $chat->getId(), $offset, 'desc', $limit, $includeLastKnown);
 	}
 
+	/**
+	 * @param Room $chat
+	 * @param int $offset Last known message id
+	 * @param array $verbs
+	 * @return IComment
+	 * @throws NotFoundException
+	 */
+	public function getNextMessageWithVerb(Room $chat, int $offset, array $verbs, bool $ignoreOffset): IComment {
```

Comment:
Should this be named `getPreviousMessageWithVerb` instead?

Reply:
fixed